spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/support/converter/AbstractMessagingMessageConverter.java

```diff
@@ -145,6 +145,13 @@ public void setHeaderMapper(HeaderMapper<S> headerMapper) {
 		this.headerMapper = headerMapper;
 	}
 
+	/**
+	 * Configure to not send TypeHeader information for {@link software.amazon.awssdk.services.sqs.model.Message} instances.
```

Comment:
On a second thought, how about something like:

`Configure the converter to not include payload type information in the message headers.`?